packages/@lightningjs/ui-components/src/components/TextBox/TextBox.test.js

```diff
@@ -168,14 +168,26 @@ describe('TextBox', () => {
       expect(textBox._Text.text.textColor).toBe(getValidColor('#000000'));
     });
 
-    // TODO: Fix reference to Base
-    // it('should get TextDefaults when switching themes', async () => {
-    //   context.setTheme(Base);
-    //   textBox.base = context.theme;
-    //   textBox.content = 'Hello Text Defaults';
-    //   await textBox.__updateSpyPromise;
-    //   expect(textBox._Text.text.maxLinesSuffix).toBe('..');
-    // });
+    it('should get TextDefaults when switching themes', async () => {
```

Comment:
By TextDefaults does it mean the [Lightning default text styles](https://lightningjs.io/docs/#/lightning-core-reference/RenderEngine/Textures/Text?id=text)? If so, would this test be testing if on theme change, text props like `fontSize` is the default size of 40 rather than updating that text style property?

Reply:
I was asking/thinking something perhaps like this:

```js
    it('should get TextDefaults when switching themes', async () => {
      const testTheme = {
        name: 'Test',
        typography: {
          body1: {
            fontFace: 'TestFont2'
          }
        }
      };
      context.setTheme(testTheme);
      textBox.content = 'Hello Text Defaults';
      await textBox.__updateSpyPromise;
      expect(context.theme.name).toBe('Test');


      const baseThemeValues = {
        typography: {
          body1: {
            fontSize: 25,
            fontWeight: 300,
            lineHeight: 40,
            verticalAlign: 'middle',
            textBaseline: 'bottom'
          }
        }
      }

      const lightningDefaultValues = {
        fontSize: 40,
        fontStyle: 'normal',
        textAlign: 'left',
        verticalAlign: 'top',
        wordWrap: true,
        maxLines: 0,
        maxLinesSuffix: '..',
        wordWrapWidth: 0
      }
      
      expect(textBox._Text.text.fontFace).toBe(testTheme.typography.body1.fontFace)
      // since we only updated the fontFace, the text props defined in the Base theme should apply to the new theme
      expect(textBox._Text.text.fontSize).toBe(baseThemeValues.typography.body1.fontSize)
      // those text props not defined in the Base theme, should use the Lightning defaults
      expect(textBox._Text.text.maxLinesSuffix).toBe(lightningDefaultValues.maxLinesSuffix);
      expect(textBox._Text.text.maxLines).toBe(lightningDefaultValues.maxLines);
    });
```

Reply:
Got it, Nice. I can add that.